File: 2025-1/sesion-de-clase-18/SoftProg/SoftProgNegocio/src/main/java/pe/edu/pucp/inf30/softprog/bo/IClienteBO.java
package pe.edu.pucp.inf30.softprog.bo;

import pe.edu.pucp.inf30.softprog.model.clientes.Cliente;

/**
 *
 * @author eric
 */
public interface IClienteBO extends IBaseBO<Cliente> {
    Cliente buscarPorDni(String dni);
}

File: 2025-2/sesion-de-clase-21/dotnet/SoftProgNegocio/BO/IClienteBO.cs
using PUCP.SoftProg.Modelo.Clientes;

namespace PUCP.SoftProg.Negocio.BO {
    public interface IClienteBO : IBaseBO<Cliente> {
        Cliente BuscarPorDni(string dni);
        Cliente BuscarPorCuenta(string cuenta);
        void Registrar(Cliente cliente);
    }
}
